kubelet/conf.yaml.example

```diff
@@ -13,11 +13,13 @@ instances:
     ### This is the default setting. See next section for legacy clusters.
     ###
     #
-    # url of the kubelet metrics endpoint
+    # url of the cadvisor metrics endpoint
     # Pass an empty string, or set the cadvisor_port option to disable
-    # prometheus metrics collection
+    # prometheus cadvisor metrics collection
     # metrics_endpoint: http://10.8.0.1:10255/metrics/cadvisor
     #
```

Comment:
Does it make more sense to have a single config option for `kubelet_api_endpoint:http://10.8.0.1:10255` and a `collect_casvisor_metrics` flag or have separate options for `kubelet_metrics_endpoint:http://10.8.0.1:10255/metrics/cadvisor` and `cadvisor_metrics_endpoint:http://10.8.0.1:10255/metrics`?

Reply:
i like the first option better, but it won't be easy to keep retro-compatibility. The second option sounds simpler. We can update the yaml with `metrics_endpoint` --> `cadvisor_endpoint` and add `kubelet_endpoint`, and fallback to `metrics_endpoint` in the check if no `cadvisor_endpoint` is found in the config object.

Reply:
IMHO the best woud be to have only `kubelet_api_endpoint` and then we can guess `/metrics` and `/metrics/cadvisor` because there's no way to change the path in the kubelet.
So I guess we can keep the current `metrics_endpoint` parse it and add paths if needed